app/templates/components/locate-me-mobile.hbs

```diff
@@ -0,0 +1,16 @@
+{{#unless findMeDismissed}}
+  <div class="find-me">
+    <button
+        onclick={{action 'locateMe'}}
```

Comment:
nit but why are we using both "locate" and "find" as verbs here?

Reply:
@julialucyhogan honestly, no idea... I looove the campsite rule, but I think we should be conservative about refactoring stuff in this app. Without rly good tests (which we lack), it almost always leads to weird regressions.

Stratgegy for us I think is to find the seams in the code, and split stuff up, but leave the code itself alone. Refactoring can come once this stuff is a little more testable.

Thoughts?